src/client/aliasviewer.py

```diff
@@ -120,22 +120,22 @@ def nick_times(self, times):
         current_times = [t for t in times if t['time'] is None]
 
         past_times.sort(key=lambda t: t['time'])
-        name_format = "<b>{}</b>"
-        past_format = "until {}"
+        name_format = "{}"
+        past_format = "{}"
         current_format = "now"
         past_strings = [(name_format.format(e['name']),
-                        past_format.format(time.strftime('%X %x', e['time'])))
+                        past_format.format(time.strftime('%Y-%m-%d &nbsp; %H:%M', e['time'])))
                         for e in past_times]
         current_strings = [(name_format.format(e['name']),
                            current_format)
                            for e in current_times]
         return past_strings + current_strings
 
     def nick_time_table(self, nicks):
-        table = '<br/><table border="0" cellpadding="0" cellspacing="0" width="250"><tbody>' \
+        table = '<br/><table border="0" cellpadding="0" cellspacing="1" width="220"><tbody>' \
```

Comment:
I changed it to 250 since I encountered some people with long aliases, and with 250 the line "The alias wasn't used by anyone else" doesn't hang to the right so awkwardly, I think. What's your opinion?

Reply:
Name and Date are now less wide, so room should be enough.
And I prefer the table to have some space on the right.
But I changed it a bit.

Reply:
Alright, sounds reasonable.